Survey/Survey/Logger.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+class Logger {
+    static let shared = Logger()
+    private init() {}
+    func log(error: Error) {
+        print("Error logged: \(error.localizedDescription)")
```

Comment:
This is not a logger. This is basically a glorified print statement. :) 

To implement a proper logger, please take a look at https://developer.apple.com/documentation/os/logging
Specifically, https://developer.apple.com/documentation/os/logging/generating_log_messages_from_your_code

Reply:
Will try to implement it